ddtrace/opentelemetry/tracer.go

```diff
@@ -0,0 +1,54 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package opentelemetry
+
+import (
+	"context"
+
+	oteltrace "go.opentelemetry.io/otel/trace"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+var _ oteltrace.Tracer = (*oteltracer)(nil)
+
+type oteltracer struct {
+	name     string
+	cfg      oteltrace.TracerConfig
+	provider *TracerProvider
+	ddtrace.Tracer
+}
+
+//todo:can we use config.SpanKind in some way
+func (t *oteltracer) Start(ctx context.Context, spanName string, opts ...oteltrace.SpanStartOption) (context.Context, oteltrace.Span) {
```

Comment:
Let's sure we have tests for each of the `oteltrace.SpanStartOption`s to verify that they are mapping over as we expect.

Reply:
Unlike Datadog options, `oteltrace.SpanStartOption`s are limited to the fields that `oteltrace.SpanConfig` has. At this stage, only spanKind was yet to be added, the rest are present in this function.